### ISSUE TITLE
Too many logs in flutter app when touching screen

### ISSUE THREAD

**Issue by kaustubhspn** [flutter/flutter#54141]
## Steps to Reproduce

1. Run `flutter run`.

**Expected results:** 

Earlier, there were not any verbose logs while touching screen, on when AppLifeCycle state changed. However, now too many logs are being logged, whenever there is a screen tap or when application transitioning  between background and foreground states.

**Actual results:**

Log on every single screen tap:

D/ColorViewRootUtil(27314): nav bar mode ignore false downX 576 downY 2057 mScreenHeight 2340 mScreenWidth 1080 mStatusBarHeight 54 globalScale 1.125 nav mode 0 rotation 0 event MotionEvent { action=ACTION_DOWN, actionButton=0, id[0]=0, x[0]=576.0, y[0]=2057.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, classification=NONE, metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=42204663, downTime=42204663, deviceId=7, source=0x1002, displayId=0 }

These is entire log when app state transitions :

D/SurfaceView(27314): 17151477 updateRequestedVisibility, mViewVisibility:true, mWindowVisibility:false, mWindowStopped:true by android.view.SurfaceView.onWindowVisibilityChanged:292 android.view.View.dispatchWindowVisibilityChanged:13970 android.view.ViewGroup.dispatchWindowVisibilityChanged:1580 android.view.ViewGroup.dispatchWindowVisibilityChanged:1580 android.view.ViewGroup.dispatchWindowVisibilityChanged:1580 android.view.ViewRootImpl.performTraversals:2333 android.view.ViewRootImpl.doTraversal:2001 android.view.ViewRootImpl$TraversalRunnable.run:8200 
D/SurfaceView(27314): 17151477 updateRequestedVisibility, mViewVisibility:true, mWindowVisibility:false, mWindowStopped:false by android.view.SurfaceView.windowStopped:264 android.view.ViewRootImpl.setWindowStopped:1722 android.view.WindowManagerGlobal.setStoppedState:685 android.app.Activity.performRestart:8058 android.app.ActivityThread.performRestartActivity:4995 android.app.servertransaction.TransactionExecutor.performLifecycleSequence:248 android.app.servertransaction.TransactionExecutor.cycleToPath:206 android.app.servertransaction.TransactionExecutor.executeLifecycleState:178 
V/SensorManager(27314): RegisterListener bmi160 Accelerometer Non-wakeup type:1 delay:66667us by com.facebook.appevents.codeless.ViewIndexingTrigger
D/SurfaceView(27314): 17151477 updateRequestedVisibility, mViewVisibility:true, mWindowVisibility:true, mWindowStopped:false by android.view.SurfaceView.onWindowVisibilityChanged:292 android.view.View.dispatchWindowVisibilityChanged:13970 android.view.ViewGroup.dispatchWindowVisibilityChanged:1580 android.view.ViewGroup.dispatchWindowVisibilityChanged:1580 android.view.ViewGroup.dispatchWindowVisibilityChanged:1580 android.view.ViewRootImpl.performTraversals:2333 android.view.ViewRootImpl.doTraversal:2001 android.view.ViewRootImpl$TraversalRunnable.run:8200 
I/SurfaceView(27314): 17151477 Changes: creating=true format=false size=false visible=true left=false top=false
I/SurfaceView(27314): 17151477 mSurfaceControl 262707974 mBackgroundControl 111805895 creating: SurfaceView - com.kaustubhpandey.lovemint/com.kaustubhpandey.lovemint.MainActivity
I/SurfaceView(27314): 17151477 Cur surface: Surface(name=null)/@0xed3c975, visible=true
I/SurfaceView(27314): 17151477 visibleChanged:true -> SurfaceHolder.Callback.surfaceCreated
I/SurfaceView(27314): 17151477 surfaceChanged -- format=4 w=1080 h=2208 visibleChanged:true -> SurfaceHolder.Callback.surfaceChanged
I/SurfaceView(27314): 17151477 surfaceRedrawNeeded
I/SurfaceView(27314): 17151477 finishedDrawing
V/SurfaceView(27314): Layout: x=0 y=0 w=1080 h=2208, frame=Rect(0, 0 - 1080, 2208)
D/SurfaceView(27314): 148145048 updateSurfacePosition RenderWorker, frameNr = 1, postion = [0, 0, 1080, 2208]
D/ColorExSystemServiceHelper(27314): checkColorExSystemService className = com.google.android.gms.measurement.service.MeasurementBrokerService

<details>
  <summary>

</summary>

```
[✓] Flutter (Channel stable, v1.12.13+hotfix.9, on Mac OS X 10.15.2 19C57,
    locale en-US)
    • Flutter version 1.12.13+hotfix.9 at
      /Users/kaustubhpandey/Developer/flutter
    • Framework revision f139b11009 (7 days ago), 2020-03-30 13:57:30 -0700
    • Engine revision af51afceb8
    • Dart version 2.7.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    • Android SDK at /Users/kaustubhpandey/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-29, build-tools 29.0.0
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C505
    • CocoaPods version 1.9.1

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 35.3.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1343-b01)

[✓] Connected device (1 available)
    • RMX1901 • 2afb0bb4 • android-arm64 • Android 10 (API 29)

• No issues found!
```

```
```

```
```

</details>

**Comment by TahaTesser** [comment]
Hi @kaustubhspn 
can you please provide your `flutter doctor -v` and `flutter run --verbose`?
Thank you

**Comment by kaustubhspn** [comment]
> Hi @kaustubhspn
> can you please provide your `flutter doctor -v` and `flutter run --verbose`?
> Also, to better address the issue, would be helpful if you could post a minimal code sample to reproduce the problem
> Thank you

Hi @TahaTesser ,

Actually this issue is similar to
 https://github.com/flutter/flutter/issues/34876#issuecomment-596332474

Also, I opened a new issue as @dnfield suggested to open a new issue regarding this. 

So, as you asked, here is output of flutter doctor -v
```
[✓] Flutter (Channel stable, v1.12.13+hotfix.9, on Mac OS X 10.15.2 19C57,
    locale en-US)
    • Flutter version 1.12.13+hotfix.9 at
      /Users/kaustubhpandey/Developer/flutter
    • Framework revision f139b11009 (7 days ago), 2020-03-30 13:57:30 -0700
    • Engine revision af51afceb8
    • Dart version 2.7.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    • Android SDK at /Users/kaustubhpandey/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-29, build-tools 29.0.0
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C505
    • CocoaPods version 1.9.1

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 35.3.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1343-b01)

[✓] Connected device (1 available)
    • RMX1901 • 2afb0bb4 • android-arm64 • Android 10 (API 29)

• No issues found!
```

I am closing this as it can be solved by downgrading to ```1.12.13+hotfix.7```
Hope it gets fixed in the next update
